integration/tests/cook/test_basic.py

```diff
@@ -79,7 +82,8 @@ def test_failing_submit(self):
         self.assertEqual('failed', job['instances'][0]['status'])
 
     def test_max_runtime_exceeded(self):
-        job_uuid, resp = self.submit_job(command='sleep 60', max_runtime=5000)
+        timeout_interval_seconds = self.settings()['task-constraints']['timeout-interval-minutes'] * 60
```

Comment:
re: `get_in`, I'm going to add it as a util function, coming soon
re: the timeout, `wait_for_job` has `stop_max_delay=120000` so the most we should wait is about 2 minutes